src/components/sdk/MeetingControls/docs/AudioInputVFControl.stories.mdx

```diff
@@ -0,0 +1,57 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import AudioInputVFControl from '../AudioInputVFControl';
+
+<Meta title="SDK Components/MeetingControls/AudioInputVFControl" />
+
+# AudioInputVFControl
+
+The `AudioInputVFControl` component renders a `ControlBarButton` with pop over menu options to select through multiple audio input options,
+and provides the option of voice focus on the bottom. If you want to apply the voice focus feature, try this `AudioInputVFControl`. 
+If not, `AudioInputControl` is a better choice. This component should be used within VoiceFocusProvider and meetingProvider. 
+Make sure you have set `enableWebAudio` as `true` in the meeting config or the voice fucos won't work.
```

Comment:
nit: fucos ->focus.

Should we say: "or the voice focus feature won't work."?

Reply:
Thanks for it, updated